libexec/compile-word-break.js

```diff
@@ -26,13 +27,6 @@ const MAX_CODE_POINT = 0x10FFFF;
 const genDirectory = path.join(__dirname, '..', 'src', 'gen');
 const generatedFilename = path.join(genDirectory, 'WordBreakProperty.ts');
 
-// Ensure this package's major version number is in sync with the Unicode
-// major version.
-const packageVersion = require('../package.json').version;
-const UNICODE_VERSION = packageVersion.split('.')[0] + '.0.0';
```

Comment:
I would still like a check SOMEWHERE that the Unicode version and the package version are consistent.